web/app/themes/justicejobs/single-job.php

```diff
@@ -90,6 +90,17 @@ class="hero__img hero__img--mobile"
                     the_content();
                     ?>
                     <h2>Additional Information</h2>
+                    <?php
```

Comment:
How comes we now adding a range here now? Was this a request?

Reply:
This was a way of getting the free text into the job advert whilst keeping the parsed salary range where it is now.  I've passed it by Anne-Marie and Marc and they want to see it in Staging.

Reply:
Great thanks